src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/Components/Account/Pages/ExternalLogin.razor

```diff
@@ -35,7 +35,7 @@
             <DataAnnotationsValidator />
             <ValidationSummary class="text-danger" role="alert" />
             <div class="form-floating mb-3">
-                <InputText @bind-Value="Input.Email" class="form-control" autocomplete="email" placeholder="Please enter your email." />
+                <InputText @bind-Value="Input.Email" id="email" class="form-control" autocomplete="email" placeholder="Please enter your email." />
```

Comment:
Can you make the id `Input.Email` that aligns with the ID that MVC generates and will be forward compatible if we ever decide to emit Ids for bindings in the future.